### PR TITLE
FTX Exchange Implemented

### PR DESCRIPTION
**Pull request by Plumbly**
This should be the FTX exchange now implemented as per #634.

### CODE REVIEW

File: src/ExchangeSharp/API/Exchanges/FTX/ExchangeFTXAPI.cs [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+		{
+			//{
+			//	"market": "XRP-PERP",
+			//  "side": "sell",
+			//  "price": 0.306525,
+			//  "type": "limit",
+			//  "size": 31431.0,
+			//  "reduceOnly": false,
+			//  "ioc": false,
+			//  "postOnly": false,
```
Comment (vslee) [inline src/ExchangeSharp/API/Exchanges/FTX/ExchangeFTXAPI.cs:372]: Looks like this exchange supports postOnly. Can you add that?

File: src/ExchangeSharp/ExchangeSharp.csproj [hunk 2]
```diff
@@ -22,6 +22,12 @@
     <RepositoryType>git</RepositoryType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="API\Exchanges\FTX\Models\**" />
```
[the remark this answers is not in the document]
Comment (Plumbly) [reply src/ExchangeSharp/ExchangeSharp.csproj]: Nope, think I accidentally added it while removing an unused folder I added.

File: src/ExchangeSharp/Model/ExchangeTicker.cs [hunk 3]
```diff
@@ -32,6 +32,11 @@ public sealed class ExchangeTicker
 		/// </summary>
 		public string Id { get; set; }
 
+		/// <summary>
+		/// The name of the exchange the tick was sent from.
+		/// </summary>
+		public string Exchange { get; set; }
```
[the remark this answers is not in the document]
Comment (Plumbly) [reply src/ExchangeSharp/Model/ExchangeTicker.cs:38]: Done. I thought I had actually done this by adding the logic on a utility method for parsing the ticker, but hadn't realised not all exchanges make use of it.

**Comment by Plumbly** [comment]
Believe that should be all the requested modifications completed.

**Review by vslee — APPROVED** [review]

**Pull request merged**